packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/ReviewWorkflows.js

```diff
@@ -122,7 +122,8 @@ export function ReviewWorkflowsPage() {
 
   useEffect(() => {
     trackUsage('didViewWorkflow');
-  }, [trackUsage]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
is `trackUsage` not stable? Currently it might be actually because it's attached to a ref (really bad code)

Reply:
It is not. I was surprised too.